packages/drawing-2d/src/symbols/window-symbol.ts

```
@@ -0,0 +1,308 @@
+          },
+        });
+        break;
+
+      case 'DOUBLE_PANEL_HORIZONTAL':
+        // Single horizontal mullion in center (along wall direction)
+        // This would be a line perpendicular to the wall
+        break;
+
+      case 'TRIPLE_PANEL_VERTICAL':
+        // Two vertical mullions
+        const thirdWidth = width / 3;
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)